Documentation/psp.md

```diff
@@ -3,6 +3,13 @@ title: Pod Security Policies
 weight: 1400
 indent: true
 ---
+{% assign url = page.url | split: '/' %}
```

Comment:
Do we need this psp.md anymore? I was thinking it would also go away with this change since the PSP will always be created with common.yaml.

Reply:
I think this could still be good for the other backends which might not have PSP info yet. Or do they all have it now?

Reply:
The page was created originally for ceph, but that's a good point to ask if the other storage providers use it. @dyusupov @yanniszark Do EdgeFS or Cassandra need this prereq for PSPs?